sklearn/utils/validation.py

```diff
@@ -1655,6 +1655,24 @@ def check_is_fitted(estimator, attributes=None, *, msg=None, all_or_any=all):
         raise NotFittedError(msg % {"name": type(estimator).__name__})
 
 
+def _estimator_has(attr, delegate_attrs=("estimator_", "estimator")):
```

Comment:
I find the param name `delegate_attrs` somehow confusing, and since we are talking about very concrete things (sub-estimators) we don't need this abstraction here. What about `delegates` instead?

Reply:
Renamed.